### ISSUE TITLE
update check mark icons?

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/sun#296]
`FontAwesomeNode.ICONS` currently has `check` and `check_without_box`. The latter is something that PhET created, and does not correspond to a Font Awesome icon; i.e.:

```js
66 // stripped the box out of the 'check'.  This was manually edited by PhET, not provided by Font Awesome directly
67 ICONS.check_without_box = "...";
```  

It looks like these could be updated to use [fa-check-square-o](http://fontawesome.io/icon/check-square-o/) and [fa-check](http://fontawesome.io/icon/check/) respectively. 

`'check'` is used exclusively by sun.CheckBox.

`'check-without-box'` is used by joist, sun, balancing-chemical-equations, graphing-lines, unit-rates.

**Comment by samreid** [comment]
Now that Font Awesome provides a check without a box, we should use it.

**Comment by pixelzoom** [comment]
Updated check icons in the above commits using font-awesome-4.7.0. 

Verified that CheckBox looks the same after switching to fa-check-square-o.

fa-check is slightly larger than PhET's check_no_box.  I verified that it looks OK in sims (all of which were mine) and in the PhET menu's "Fullscreen" menu item (the only place it appears in common code).

Leaving this labeled for developer meeting to alert developers to this change. Then I'll close it.
